src/rust/examples/slapper/src/main.rs

```diff
@@ -0,0 +1,61 @@
+use std::path::PathBuf;
+
+use clap::{Parser, Subcommand};
+
+mod commands;
+pub mod requests;
+
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+struct Cli {
+    /// Optional color to test
+    #[arg(short, long)]
+    color: Option<String>,
+
+    /// Optional language to test
+    #[arg(short, long)]
+    language: Option<String>,
+
+    /// Iterations to test
+    #[arg(short, long)]
+    iterations: Option<i32>,
```

Comment:
I think `iterations` should be an unsigned type to disallow use of negative numbers. For example `usize`, or `u32`.